src/os.toit

```diff
@@ -56,6 +58,16 @@ A map-like object that represents the environment variables of the current
 */
 env / EnvironmentVariableMap ::= EnvironmentVariableMap.private_
 
+/**
+The path separator for the underlying operating system.
+*/
+path-separator -> string:
```

Comment:
This is typically called "List separator".
I would combine Posix with Freertos. If an env-variable had multiple entries, then they should be separated by `:` on freertos. imho.

Also not sure if it should be in the os library. I'm adding it right now to the fs package.

Reply:
It's arguably about environment variables, not filenames.

If it's about filenames, then it should not be available on Freertos, since we don't support PATH there, and never will.